scripts/projects.js

```diff
@@ -3,12 +3,14 @@ var rawData = [
     title:       'Guac A Mole Didactic Tool',
     category:    'Didactic',
     publishedOn: '2016-01-29',
-    body:        '<p>Guac A Mole is a didactic game/tool applicable to different learning categories. The user engages with a series of empty spaces, populated with question/answer combos using setInterval methods and objects built using a JS constructor. The user must clear these boxes by answering them before too many new ones come up. The user can then look at their results and play again as they wish </p>'
+    body:        '<p>Guac A Mole is a didactic game/tool applicable to different learning categories. The user engages with a series of empty spaces, populated with question/answer combos using setInterval methods and objects built using a JS constructor. The user must clear these boxes by answering them before too many new ones come up. The user can then look at their results and play again as they wish </p>',
+    image: '<section class="project-image"><img src="images/guacamole.png" class="project-image-img" /></section>'
```

Comment:
Normally you want to save just the image URL here and insert that into the element's attribute when you create it.